superbench/executor/executor.py

```diff
@@ -87,6 +87,10 @@ def __get_platform(self):
                 return Platform.CUDA
             elif gpu.vendor == 'amd':
                 return Platform.ROCM
+            elif gpu.vendor == 'amd-win':
+                return Platform.DIRECTX
+            elif gpu.vendor == 'nvidia-win':
+                return Platform.DIRECTX
```

Comment:
Do we need to separate AMD or Nvidia? If so, we should separate them here as well. Otherwise, why do we keep amd-win and nvidia-win?

Reply:
separated